### PR TITLE
feat: Contact Form for Contact Us page and Input component

### PR DESCRIPTION
**Pull request by tamrivera05**

### CODE REVIEW

**Comment by tamrivera05** [comment]
* **#17** <a href="https://app.graphite.dev/github/pr/TimBuilding/Elysseum/17?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#16** <a href="https://app.graphite.dev/github/pr/TimBuilding/Elysseum/16?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#15** <a href="https://app.graphite.dev/github/pr/TimBuilding/Elysseum/15?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#8** <a href="https://app.graphite.dev/github/pr/TimBuilding/Elysseum/8?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#7** <a href="https://app.graphite.dev/github/pr/TimBuilding/Elysseum/7?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @tamrivera05 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: src/pages/contact.astro [hunk 1]
```diff
@@ -1,8 +1,27 @@
 ---
 import PageLayout from "../layouts/PageLayout.astro";
+import ContactForm, {ContactFormProps} from "../components/ContactFormProps";
 
+const contacts: ContactFormProps[] = [
```
Comment (xdjeremy) [inline src/pages/contact.astro]: Use array instead, if title lang naman
```javascript
const contacts = ['Name', 'Phone Number', 'etc'] 
```

**Review by xdjeremy — CHANGES_REQUESTED** [review]

File: src/pages/contact.astro (hunk 1 above)
Comment (tamrivera05) [reply src/pages/contact.astro]: oki po mwa

**Review by xdjeremy — APPROVED** [review]
[cool](https://app.graphite.dev//settings/meme-library?org=TimBuilding)

**Pull request merged**